tools/clang/test/DXC/Passes/DxilGen/tracerayinline_dxilgen.ll

```diff
@@ -0,0 +1,132 @@
+; RUN: %dxopt %s -hlsl-passes-resume -dxilgen -S | FileCheck %s
+
+; Based on tools/clang/test/CodeGenDXIL/hlsl/objects/RayQuery/tracerayinline.hlsl,
+; with call to DoTrace commented out.
```

Comment:
Just to confirm, the inclusion of DoTrace didn't make this test impossible, but it just added more generated code that wasn't necessary to the test either?

Reply:
Correct, it would have only doubled some of the code for no benefit, making it harder to test.

Reply:
nit: That would be helpful information to include in the comment. "with call DoTrace commented out as it adds unnecessary generated code".